pyproject.toml

```diff
@@ -1,32 +1,107 @@
 [build-system]
 requires = [
-    "jupyter_packaging~=0.12,<2", 
-    "jupyterlab==4.0.0"
+    "hatchling",
+    "jupyterlab==4.0.0",
 ]
-build-backend = "jupyter_packaging.build_api"
+build-backend = "hatchling.build"
 
-[tool.jupyter-packaging.options]
-skip-if-exists = [
-    "jupyterlab_latex/labextension/static/style.js"
+[project]
+name = "jupyterlab-latex"
+description = "JupyterLab extension for running LaTeX"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    { name = "Jupyter Development Team", email = "jupyter@googlegroups.com" },
+]
+keywords = [
+    "Jupyter",
+    "JupyterLab",
+    "LaTeX",
+]
+classifiers = [
+    "Framework :: Jupyter",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "jupyter_server>=2,<3",
+    "jupyterlab>=4,<5",
```

Comment:
I would drop the dependency on JupyterLab itself (except if you call a specific endpoint). This will allow to install this extension in Notebook v7 without installing JupyterLab as I guess this extension does not depend directly on the JupyterLab layout.

```suggestion
```